DirectEmployers/MyJobs
# [PD-3664] fix header in export

## What's this PR do?
Start using `alias` in excel/csv exports, this includes headers and filter criteria.

## Screenshots (if appropriate)
Before:

![image](https://user-images.githubusercontent.com/27963853/30296267-9c5e8f20-9711-11e7-95f2-85d4c5d597e3.png)

After:

![image](https://user-images.githubusercontent.com/27963853/30296226-6dfb834a-9711-11e7-9f78-802e98a62b0e.png)

## How should this be manually tested?
Export for different types, both excel and csv should have their headers set properly. For excel, the criteria sheet should also have correct labels.

## Any background context you want to provide?
@tejaswinigutti31 and I decided to refactor a bit, and renamed `label` to `alias` for columns defined in schema_dimensions.py - `alias` was introduced in an old commit, we'll let it live, and let's agree on its meaning and purpose: "label for a column, something easier to read than snake case, used for front end representation"
